src/test/resources/spec/concordion/command/example/Example.html

```diff
@@ -91,6 +101,60 @@
         <span c:assertTrue="beforeSpecRunOnce()">exactly once</span> before
         any examples are run. Methods annotated with @AfterSpecification are executed after the specification has executed.</p>
 
+    <p>Finally, we assert that <span c:assertEquals="numberTableRowsInserted()">2</span> table rows were added to our database.</p>
+
+</div>
+
+
+<div class="example">
+    <h3>Tables</h3>
+
+    <p>
+        Concordion can execute each row in a table as an example. To do this, use the c:example tag in one of the table header rows.
+    </p>
+
+    <p>For example:</p>
+
+    <code><pre>
+    &lt;table c:execute="#counter = incrementCounter()"&gt;
+        &lt;tr&gt;
+            &lt;th c:example="#TEXT"&gt;Example Name&lt;/th&gt;
```

Comment:
This doesn't need to be `#TEXT` and is somewhat confusing having it there. An empty attribute will do, although we seem to need to specify `c:example=""` rather than `c:example`